### PR TITLE
Add Ethereum canister to the showcase

### PR DESCRIPTION
**Pull request by zvolin**
Hi, as requested in the final demo checklist, I'm submitting the ethereum canister to the showcase.json. As it's yet a poc that has no real users, no logo etc I left a few fields empty

Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by zvolin** [comment]
> Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

it seems I cannot retrigger

File: showcase.json [hunk 1]
```diff
@@ -2419,5 +2419,23 @@
...
+  },
+  {
+    "id": "ethereum-canister",
+    "name": "Ethereum Canister",
+    "website": "https://www.eiger.co/",
+    "oneLiner": "A fully trustless access to the Ethereum blockchain data.",
+    "tags": [
+      "Ethereum", "Tools / Infrastructure",
+    ],
+    "description": "The Ethereum canister offers a secure and trustless way to access Ethereum blockchain data within the ICP ecosystem. Behind the scenes, it leverages the helios light Ethereum clien which is equipped with the capability to validate the authenticity of fetched data.",
```
Comment (letmejustputthishere) [inline showcase.json:2431]: can you fix `clien`

Comment (zvolin) [reply showcase.json]: done

File: showcase.json [hunk 2]
```diff
@@ -2419,5 +2419,23 @@
...
     "screenshots": ["/img/showcase/motokopilot_screenshot.jpg"],
     "submittableId": "35960849"
+  },
+  {
+    "id": "ethereum-canister",
+    "name": "Ethereum Canister",
+    "website": "https://www.eiger.co/",
+    "oneLiner": "A fully trustless access to the Ethereum blockchain data.",
+    "tags": [
+      "Ethereum", "Tools / Infrastructure",
```
Comment (letmejustputthishere) [inline showcase.json:2429]: pls remove the  trailing comma

**Review by letmejustputthishere — CHANGES_REQUESTED** [review]

File: showcase.json (hunk 2 above)
Comment (zvolin) [reply showcase.json]: done

**Review by letmejustputthishere — APPROVED** [review]

**Pull request merged**